portfolio/src/main/java/com/google/sps/data/ShortestTaskFirstScheduler.java

```diff
@@ -39,13 +39,12 @@
       Instant workHoursEndTime) {
     List<CalendarEvent> eventsList = new ArrayList<CalendarEvent>(events);
     List<Task> tasksList = new ArrayList<Task>(tasks);
-    Collections.sort(tasksList, sortByTaskDurationThenName);
     CalendarEventsGroup calendarEventsGroup =
         new CalendarEventsGroup(eventsList, workHoursStartTime, workHoursEndTime);
     List<TimeRange> availableTimes = calendarEventsGroup.getFreeTimeRanges();
+    TaskGroup taskGroup = new TaskGroup(tasksList, getSchedulingAlgorithmType());
```

Comment:
See my comment below on changing these to TaskQueue, taskQueue, etc.

Reply:
Done.